src/ConfigFactory.Core/ConfigModule.cs

```diff
@@ -74,16 +74,16 @@ private static T Load()
 
     public virtual void Reset()
     {
-        IConfigModule config = Load();
-        foreach ((var name, (var property, _)) in Properties) {
+        var config = Load();
```

Comment:
Prefer explicit type names

```suggestion
        IConfigModule config = Load();
```